client/controllers/incidentReports/incidentTable.coffee

```diff
@@ -0,0 +1,33 @@
+Template.incidentTable.onCreated ->
+  @rowsSelected = new ReactiveVar(false)
+  @selectedIncidents = new ReactiveVar([])
+
+changeIncidentStatus = (status, instance) ->
+  $.each instance.selectedIncidents.get(), (index, incidentId) ->
+    incident = _id: incidentId
+    incident.accepted = status
+    Meteor.call 'updateIncidentReport', incident, false, (error, result) ->
+      if error
+        console.log "ERROR calling editIncidentReport", error
+        toastr.error("There was a problem updating your incident reports!")
+        return
+      toastr.success("Incidents reports udpated!")
+
+Template.incidentTable.helpers
+  showMultiActions: ->
+    Template.instance().rowsSelected.get()
```

Comment:
I think we can avoid tracking this with its own reactiveVar and within this helper check if `selectedIncidents` has a length.